.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,28 @@
+name: Feature Request
```

Comment:
Feature Requests makes little sense when this repo does not hold software.
This repo is purely documentation.
The term "feature request" doesn't really apply here.